models.py

```diff
@@ -107,3 +107,52 @@ class CCRecruitment(BaseModel):
         str_strip_whitespace=True,
         validate_assignment=True,
     )
+
+
+class Base64Validator(BaseModel):
+    base64_bytes: Optional[Base64Bytes] = None
+    base64_str: Base64Str
+
+
+class StorageFile(BaseModel):
+    id: PyObjectId = Field(default_factory=PyObjectId, alias="_id")
+    title: str = ""
+    data: Base64Str | None = None
+    filetype: str = "pdf"
+    modified_time: str = ""
+    created_time: str = ""
+
+    @field_validator("data")
+    @classmethod
+    def validate_filedata(cls, value):
+        Base64Validator(base64_str=value).base64_str
+        return value
+
+    model_config = ConfigDict(
+        populate_by_name=True,
+        arbitrary_types_allowed=True,
+        extra="forbid",
+        str_strip_whitespace=True,
+        validate_assignment=True,
+    )
+
+class PyInputStorageFileDetails(BaseModel):
```

Comment:
No need of a separate class. In the inputs type which you have created in otypes, just specify which fields are to be accepted. 

Completely different model creation for the same thing brings in duplication, we should try to avoid it

Reply:
You need to derive a class from PyDantic, since strawberry by default doesn't recognize Base64Str, i was getting schema export, error if I didn't use @strawberry.experimental.pydantic.input

Reply:
I have pushed a new commit to even fix the issue you mentioned